components/src/core/ChannelValue/ChannelValue.tsx

```diff
@@ -95,15 +126,18 @@ const ChannelValueRender: React.ForwardRefRenderFunction<unknown, ChannelValuePr
                     <Typography
                         variant={'h6'}
                         color={'inherit'}
-                        className={clsx(defaultClasses.text, defaultClasses.units, classes.units)}
+                        className={clsx(defaultClasses.text, defaultClasses.units, classes.units, {
+                            [defaultClasses.prefix]: prefix && applyPrefix(),
```

Comment:
Can you move the `prefix &&` to inside the `applyPrefix` function?  It was a little confusing for me to read at first.

Reply:
Done please review https://github.com/brightlayer-ui/react-component-library/pull/362/commits/09589f583a1fb48c662c2e0af95f7e886ab940fe#diff-9b02b302c55684c472c77ad7327b46f3aa673c8fbda241554b4a144edd0eeacdR111